.github/workflows/ci.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: ['8', '11', '15']
+      
+    steps:
+    - uses: actions/checkout@v2
+    - name: ${{ matrix.java }}
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+
+    - name: Maven Install Dependencies
+      run: mvn install -B -V -DskipTests -Dair.check.skip-all
+    - name: Maven Offline
+      run: mvn dependency:go-offline -B -Dsilent
```

Comment:
Why this?

cc @wendigo

Reply:
Would that solve https://github.com/prestosql/presto/issues/5816 ?

Reply:
We can try that @findepi